app/fellow.py

```diff
@@ -0,0 +1,63 @@
+from os import sys, path
+
+from app.person import Person
+
+sys.path.append(path.dirname(path.dirname(
+    path.dirname(path.abspath(__file__)))))
+
+
+class Fellow(Person):
+    """docstring for Fellow"""
+
+    fellow_list = []
+    unallocated_fellows = {"office": [], "livingspace": []}
```

Comment:
it holds the unallocated fellows. Those yet to have an office will be appended to the office section of the dictionary, while those yet to get a livingspace will be appended to the livingspace section of the dictionary and those yet to get both office and livingspace will be appended to both sections of the dictionary.